source/client/menu.qc

```diff
@@ -983,6 +989,7 @@ var struct
 	// Control Settings
 	{[6, 75], "ADS Mode", -1, settings_adsmode, null, MENU_CONSETTINGS, 0, OPTION_WEB_AND_EXE}, // 54
 	{[6, 95], "Look Sensitivity", -1, settings_sens, settings_sens2, MENU_CONSETTINGS, 0, OPTION_WEB_AND_EXE}, // 55
+	{[6, 155], "Rumble", -1, settings_rumble, null, MENU_CONSETTINGS, 0, OPTION_WEB_AND_EXE}, // 55
```

Comment:
Don't do this. Add to the bottom of the list because menu descriptions use indexing to identify the highlighted button.

Reply:
I skipped it because I was too lazy to reposition the option for the controller glyphs. It seems to work okay?

Reply:
You won't have to reposition anything. The drawn position of the element is fine, just move it further down the array so it does not offset menu option descriptions

Reply:
Oh, you said descriptions. My bad, that is wrong.